public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -141,7 +141,7 @@ const getStyles = (theme: GrafanaTheme2, hasActions: boolean) => {
     content: css({
       display: 'flex',
       flexDirection: 'column',
-      paddingTop: hasActions ? TOP_BAR_LEVEL_HEIGHT * 2 : TOP_BAR_LEVEL_HEIGHT,
+      paddingTop: hasActions ? TOP_BAR_LEVEL_HEIGHT : 0,
```

Comment:
I think here and in AppChromeService we should have a `/* (NI Fork) */` comment describing the change and intention.